source/style-guide/information-types/prototypes/reference/server-params-style.txt

```diff
@@ -0,0 +1,129 @@
+.. _meta-reference-server-params:
+
+=======================
+Server Parameters Style
```

Comment:
[s] For consistency with the naming of other pages in the style guide, I would suggest: 
```suggestion
Server Parameters Guidelines
```

Reply:
Thanks for this suggestion. Updated!